draft-ietf-privacypass-architecture.md

```diff
@@ -436,203 +415,178 @@ clients cannot determine if this value is correct or otherwise a tracking
 vector.
 
 Private metadata is that which clients cannot observe as part of the token
-issuance flow. In {{?I-D.ietf-privacypass-protocol}}, it is possible to include
-private metadata to redemption tokens. The core protocol instantiation that
-is described does not include additional metadata. However, future instantiations
-may use this functionality to provide redemption verifiers with additional
-information about the user. Such instantiations may be built on the Private
-Metadata Bit construction from Kreuter et al. {{?KLOR20=DOI.10.1007/978-3-030-56784-2_11}}
+issuance flow. Such instantiations may be built on the Private Metadata Bit
+construction from Kreuter et al. {{?KLOR20=DOI.10.1007/978-3-030-56784-2_11}}
 or the attribute-based VOPRF from Huang et al. {{HIJK21}}.
 
 Metadata may also be arbitrarily long or bounded in length. The amount of
 permitted metadata may be determined by application or by the underlying
 cryptographic protocol.
 
-## Client privacy implications
+### Extensibility {#extensions}
 
-Note that any metadata bits of information can be used to further
-segment the size of the user's anonymity set. Any server that wanted to
-track a single user could add a single metadata bit to user tokens. For
-the tracked user it would set the bit to `1`, and `0` otherwise. Adding
+The Privacy Pass protocol and ecosystem are both intended to be
+receptive to extensions that expand the current set of functionality.
+All extensions to the Privacy Pass protocol SHOULD be specified as separate
+documents that modify the content of this document in some way.
```

Comment:
I don't understand this requirement -- why SHOULD extensions modify this document? Also, there's a Redemption protocol and an Issuance protocol -- there is no mention of the Privacy Pass protocol so far in this doc.

Reply:
Rephrased!